cnames_active.js

```diff
@@ -2731,6 +2731,7 @@ var cnames_active = {
   "spacebot": "spacebot2.github.io",
   "spark": "hammerheadstudios.github.io/SparkCode",
   "spark-handler": "cname.vercel-dns.com", // noCF
+  "sparkoj": "winterant.github.io/OnlineJudge", // SparkOJ
```

Comment:
```suggestion
  "sparkoj": "winterant.github.io/OnlineJudge",
```
Your entry should not have an arbitrary comment after it